packages/react-select/src/components/Menu.tsx

```diff
@@ -317,6 +318,7 @@ export const MenuPlacer = <
 ) => {
   const {
     children,
+    isLoading,
```

Comment:
@labkey-nicka I'm not a maintainer but while fixing the same issue I found that you can get the isLoading property from `props.selectProps.isLoading`. Therefore, we might not need an extra `isLoading` prop here.

Reply:
Hey, thanks! Indeed `isLoading` is available on `props.selectProps`. Refactored and updated.